action.yml

```diff
@@ -87,6 +83,6 @@ runs:
         GHA_PIP_AUDIT_EXTRA_INDEX_URLS: "${{ inputs.extra-index-urls }}"
         GHA_PIP_AUDIT_IGNORE_VULNS: "${{ inputs.ignore-vulns }}"
         GHA_PIP_AUDIT_INTERNAL_BE_CAREFUL_ALLOW_FAILURE: "${{ inputs.internal-be-careful-allow-failure }}"
-        GHA_PIP_AUDIT_INTERNAL_BE_CAREFUL_DEBUG: "${{ inputs.internal-be-careful-debug }}"
         GHA_PIP_AUDIT_INTERNAL_BE_CAREFUL_EXTRA_FLAGS: "${{ inputs.internal-be-careful-extra-flags }}"
+        ACTIONS_STEP_DEBUG: "${{ secrets.ACTIONS_STEP_DEBUG }}"
```

Comment:
We can drop this entirely and check for `RUNNER_DEBUG=1` in the action; GitHub will (should?) set that environment variable when `secrets.ACTION_STEP_DEBUG` is set.